### PR TITLE
Feature/#9 automatic account creation

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: .github/workflows/reusable.yaml [hunk 1]
```diff
@@ -70,6 +76,14 @@ jobs:
...
           body-includes: 'Uffizzi Preview ID:'
           direction: last
+      - name: Obtain an OIDC token from GHA
+        id: oidc-token
+        run: |
+          OIDC_TOKEN=$(curl -sLS "${ACTIONS_ID_TOKEN_REQUEST_URL}&audience=uffizzi" -H "User-Agent: actions/oidc-client" -H "Authorization: Bearer $ACTIONS_ID_TOKEN_REQUEST_TOKEN")
+          echo "::set-output name=id_token::${OIDC_TOKEN}"
+
+      - name: Check output of the token
+        run: echo "${{ steps.oidc-token.outputs.id_token }}"
```
Comment (axisofentropy) [inline .github/workflows/reusable.yaml:86]: This probably appears as `***` right?

Comment (moklidia) [reply .github/workflows/reusable.yaml]: Yeah, just was checking that it exists there

File: action.yaml [hunk 2]
```diff
@@ -11,31 +11,43 @@ inputs:
...
+    required: false
 outputs:
   url:
     description: 'URL where preview was deployed, if successful'
   id:
     description: 'ID of prevew for later reference'
 runs:
   using: 'docker'
-  image: 'docker://uffizzi/cli:v0.11.4'
+  image: 'docker://limoha/cli:latest'
```
Comment (zipofar) [inline action.yaml:50]: Does not forget change it to `uffizzi/cli` I guess

File: .github/workflows/reusable.yaml [hunk 3]
```diff
@@ -151,7 +172,7 @@ jobs:
           reactions: rocket
 
       - name: Update Existing Preview
-        uses: UffizziCloud/update-preview-action@v1
+        uses: UffizziCloud/update-preview-action@feature/#76-update-by-oidc-token
```
Comment (zipofar) [inline .github/workflows/reusable.yaml:175]: Does not forget change it to correct branch

File: .github/workflows/reusable.yaml [hunk 4]
```diff
@@ -81,13 +96,15 @@ jobs:
 
       - name: Delete Preview from Uffizzi
         if: ${{ github.event.action == 'closed' && steps.parse-comment.outputs.id != '' }}
-        uses: UffizziCloud/delete-preview-action@v1.0.0
+        uses: UffizziCloud/delete-preview-action@feature/#76_authentication_from_oidc_token
```
Comment (zipofar) [inline .github/workflows/reusable.yaml:99]: Does not forget change it to correct branch

**Pull request merged**